.github/ISSUE_TEMPLATE/release.md

```diff
@@ -4,98 +4,147 @@ about: Instructions and checklist for creating a release.
 title: 'Release X.Y.Z-rc.N'
 labels: kind/release
 assignees: ''
-
 ---
 
-# Release {version}
+# Open Match Release Process
+
+Follow these instructions to create an Open Match release.  The output of the
+release process is new images and new configuration.
+
+## Getting setup
+
+*note: the commands below are pasted from the 0.5 release.  make the necessary
+changes to match your naming & environment.*
+
+The Git flow for pushing a new release is similar to the development process
+but there are some small differences.
+
+**1. Clone your fork of the Open Match repository.**
+
+```shell
+git clone git@github.com:afeddersen/open-match.git
+```
+**2. Move into the new open-match directory.**
+
+```shell
+cd open-match
+```
+
+**3. Configure a remote that points to the upstream repository. This is required to sync changes you make in a fork with the original repository.  Note: Upstream is the gatekeeper of the project or the source of truth to which you wish to contribute.**
+
+```shell
+git remote add upstream https://github.com/GoogleCloudPlatform/open-match.git
+```
+
+**3. Fetch the branches and their respective commits from the upstream repo.**
+
+```shell
+git fetch upstream
+```
 
-<!--
-This is the release issue template. Fill in relevent values, found inside {}
-{version} should be replaced with the version ie: 0.5.0.
+**4.  Create a local release branch that tracks upstream and check it out.**
 
-There are 3 types of releases:
- * Release Candidates - 1.0.0-rc.1
- * Full Releases - 1.2.0
- * Hot Fixes - 1.0.1
+```shell
+git checkout -b release-0.5 upstream/release-0.5
+```
 
-# Release Candidate and Full Release Process
+## Releases & Versions
```

Comment:
Would it make more sense to have the version string established at the top?
In the previous steps you need to know what version strings you are going to be dealing with.
There's the `release-X.Y` branch and the `vX.Y-rcZ` version strings you need to know before starting.

Reply:
I think the syntax is more important but I'm not that opinionated about it.  If you don't feel strongly, and it's correct, let's leave it as-is.